config/locales/en.yml

```diff
@@ -875,6 +884,11 @@ en:
           error: "The following list of samples failed to transfer:"
           no_samples_transferred_error: "Samples were not transferred for the following reasons:"
       metadata:
+        edit:
+          edit: Edit
```

Comment:
**suggestion:** Can you change the label of the button to open the modal, and the button to save the updated metadata to `Update`.

Reply:
Updated in 37288a651a4b8797332102de5ea22a1996b006d4